src/rs.cpp

```diff
@@ -1248,31 +1249,19 @@ HANDLE_EXCEPTIONS_AND_RETURN(0, RS2_API_MAJOR_VERSION, RS2_API_MINOR_VERSION, RS
 
 rs2_context* rs2_create_recording_context(int api_version, const char* filename, const char* section, rs2_recording_mode mode, rs2_error** error) BEGIN_API_CALL
 {
-    VALIDATE_NOT_NULL(filename);
-    VALIDATE_NOT_NULL(section);
-    verify_version_compatibility(api_version);
-
-    return new rs2_context{ std::make_shared<librealsense::context>(librealsense::backend_type::record, filename, section, mode) };
+    throw not_implemented_exception( "deprecated" );
```

Comment:
This answers one of my previous comments, but please add there the fact that exception is thrown when calling the deprecated api methods.

Reply:
No. A deprecated method should not be called. If someone does, he get an exception.

Reply:
ok